frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -163,10 +180,19 @@ export const AccountsSummary = ({
     accounts.forEach(account => {
       const currentCode = account.code;
       subscriptions.push(statusChanged(account.code, () => currentCode === account.code && update(account.code)));
-      subscriptions.push(syncdone(account.code, () => currentCode === account.code && update(account.code)));
+      subscriptions.push(syncdone(account.code, () => {
+        if (currentCode === account.code) {
+          update(account.code);
+        }
+        getAccountSummary();
```

Comment:
getAccountSummary is already called by `update()`, so this is redundant

Reply:
Good catch, thanks!